### PR TITLE
164/changesets

### LINKED ISSUE #164: Prepare Changelog
Bartek, feel free to introduce a tool for this

### PR DESCRIPTION
**Pull request by bartmacbartek**
Closes #164 

**TODO before merge:**
- [x] mark `ci/circleci: changeset-validation` status check as required
- [x] accept [changeset-bot](https://github.com/apps/changeset-bot) installation request (should be on org email)
  - it'll provide nice comments about missing changesets with additional interface to add it from within the PR

@ethanfrey 🙏

### CODE REVIEW

File: .changeset/stupid-oranges-tie.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@confio/relayer': patch
+---
+
+Setup changesets to keep the changelog up to date.
```
Comment (bartmacbartek) [inline .changeset/stupid-oranges-tie.md:5]: The changeset filename is automatically generated.

When a release happens, all the changesets are collected to create a nice release note in the `CHANGELOG.md` and in GitHub releases. Then the changesets are pruned.

**Comment by ethanfrey** [comment]
Approved changeset bot

**Comment by ethanfrey** [comment]
> mark `ci/circleci: changeset-validation` status check as required

Where is this set? I can't find the place in github or in the circle ci config file.

**Review by ethanfrey — APPROVED** [review]
Looks good.

Tried it out manually and I like it.
Please add `yarn changeset version` to the readme, as what we need to to do generate a new version/changeset.

Otherwise, just let me know what I need to do to mark this as a required CI check (I have never seen optional ones before, I think they always are required)

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -143,6 +143,49 @@ have set the desired Cosmos SDK tag there (eg. `v0.41.0`). After that, run:
 This will overwrite the data in `src/codec` with newly generated definitions. We delete the folder first
 to avoid outdated artifacts, meaning any manual changes will be lost.
 
+### Maintain a clean changelog
+We use [changesets](https://github.com/atlassian/changesets) to keep track of changes between the releases. A changeset is a markdown file that describes changes made. Every pull request must contain a changeset.
```
Comment (willclarktech) [inline DEVELOPMENT.md:147]: In CosmJS we format `.md` files with prettier.

**Review by willclarktech — APPROVED** [review]
Never used this before but interested to see how it works out.

**Comment by ethanfrey** [comment]
Nice message.

**Pull request merged**